arcadebackend/index.js

```diff
@@ -1,20 +1,23 @@
 import express, {json} from "express"
 import { exec } from "child_process"
+import cors from "cors"
+import path from "path"
 
 const app = express()
 
-const PORT = 3000
+const PORT = 3001
 
 app.use(json())
+app.use(cors());
+
+process.chdir("C:/Users/KoksR2/Desktop/third-year-studio-2024-rainy-arcade-2"); //Set the directory
```

Comment:
this wont work on a computer that is logged in as someone else so the file path would be ./Desktop/third-year-studio-2024-rainy-arcade-2